bliss/report.py

```diff
@@ -0,0 +1,102 @@
+"""Functions to generate metrics on real data."""
+import torch
```

Comment:
Would it be OK to include the code from metrics.py and plot.py into report.py (perhaps better called reporting.py), so we can delete those two files, and keep all the reporting code together?

Reply:
Yes, I like this idea